src/parser/jobs/ast/modules/Divination.tsx

```diff
@@ -0,0 +1,297 @@
+import {t} from '@lingui/macro'
+import {Plural, Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {RotationTable} from 'components/ui/RotationTable'
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {EventHook} from 'parser/core/Dispatcher'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {Actor, Actors} from 'parser/core/modules/Actors'
+import {Data} from 'parser/core/modules/Data'
+import Suggestions, {SEVERITY, Suggestion} from 'parser/core/modules/Suggestions'
+import {Timeline} from 'parser/core/modules/Timeline'
+import React, {Fragment} from 'react'
+import {Icon, Message} from 'semantic-ui-react'
+import DISPLAY_ORDER from './DISPLAY_ORDER'
+
+const BASE_GCDS_PER_WINDOW = 6
+const PLAYERS_BUFFED_TARGET = 8
+
+interface DIVINATION_Window {
```

Comment:
style nit:
```suggestion
interface DivinationWindow {
```

Reply:
updated both this one and lightspeed with the same comment